relayer/src/event/rpc.rs

```diff
@@ -59,52 +59,68 @@ pub fn get_all_events(
 pub fn build_event(mut object: RawObject) -> Result<IbcEvent, EventError> {
     match object.action.as_str() {
         // Client events
-        "create_client" => Ok(IbcEvent::from(ClientEvents::CreateClient::try_from(
-            object,
-        )?)),
-        "update_client" => Ok(IbcEvent::from(ClientEvents::UpdateClient::try_from(
-            object,
-        )?)),
-        "submit_misbehaviour" => Ok(IbcEvent::from(ClientEvents::ClientMisbehaviour::try_from(
-            object,
-        )?)),
+        "create_client" | ibc::ics02_client::msgs::create_client::TYPE_URL => Ok(IbcEvent::from(
+            ClientEvents::CreateClient::try_from(object)?,
+        )),
+        "update_client" | ibc::ics02_client::msgs::update_client::TYPE_URL => Ok(IbcEvent::from(
+            ClientEvents::UpdateClient::try_from(object)?,
+        )),
+        "submit_misbehaviour" | ibc::ics02_client::msgs::misbehavior::TYPE_URL => Ok(
```

Comment:
@ancazamfir I remember some TODO or a loose end with respect to misbehavior/evidence events or handling. Any thoughts?

Reply:
yes, this was wrt to testing misbehaviour which we don't cover in CI. Just tested now, all looks good!